examples/large_models/inferentia2/llama2/Readme.md

```diff
@@ -78,7 +78,7 @@ huggingface-cli login
 
 Run the `inf2_save_split_checkpoints.py` script
 ```bash
-python ../util/inf2_save_split_checkpoints.py --model_name meta-llama/Llama-2-13b-hf --save_path './llama-2-13b-split'
+python ../util/inf2_save_split_checkpoints.py --model_name meta-llama/Llama-2-13b-hf --save_path './llama-2-13b-split' generate_neuron_cache --neuron_cache_dir './neuron_cache' --batch_size 4 --amp 'bf16' --tp_degree 6
```

Comment:
nit: can you link to some official docs describing what the tp degree means?

Reply:
This section of the Neuron documentation has a description of what `tp_degree` means
https://awsdocs-neuron.readthedocs-hosted.com/en/latest/libraries/neuronx-distributed/api_guide.html?highlight=tp_degree#model-trace